controller/AbstractRestQti.php

```diff
@@ -233,4 +238,13 @@ private function getMetadataService(): MetadataService
     {
         return $this->getPsrContainer()->get(MetadataService::SERVICE_ID);
     }
+
+    protected function getQueryParams($key): mixed
```

Comment:
can we use ?string here? Looks like we can

Reply:
it can be also `GuzzleHttp\Psr7\UploadedFile` So I would prefer to have mix